test/src/test/java/org/corfudb/integration/LockIT.java

```diff
@@ -60,55 +61,55 @@
                     .setLockGroup(LOCK_GROUP)
                     .setLockName(LOCK_NAME)
                     .build();
 
-           // Since this is the only client, lock should've been acquired, verify, block until condition is met
-           System.out.println("***** Wait until lock is acquired");
-           waitCondition = WaitConditionType.LOCK_ACQUIRED;
-           blockUntilWaitCondition.acquire();
-
-           int lockCount = 1;
-           assertThat(lockAcquiredObservables.get(clientId).getValue()).isEqualTo(lockCount);
-           assertThat(client.getLocks().get(lockId).getState().getType()).isEqualTo(LockStateType.HAS_LEASE);
-           assertThat(lockRevokedObservables.get(clientId).getValue()).isEqualTo(0);
-
-           waitCondition = WaitConditionType.NONE;
-
-           // Verify for 5 cycles that the lock is renewed
-           for (int i=0; i < RENEW_CYCLES; i++) {
-               System.out.println("***** Wait until lock is renewed");
-               // TODO: inspect the     // how many times the lease has been acquired by a different client
-               //    int32 lease_acquisition_number = 3;
-               //    // how many times the lease has been renewed since the last acquisition
-               //    int32 lease_renewal_number = 4;
-               // Wait for the renewal cycle + 1, and verify that the lock is still acquired
-               Sleep.sleepUninterruptibly(Duration.ofSeconds(LOCK_TIME_CONSTANT + 1));
-               assertThat(lockAcquiredObservables.get(clientId).getValue()).isEqualTo(lockCount);
-               assertThat(client.getLocks().get(lockId).getState().getType()).isEqualTo(LockStateType.HAS_LEASE);
-               assertThat(lockRevokedObservables.get(clientId).getValue()).isEqualTo(0);
-           }
-       } catch (Exception e) {
-           System.out.println("Unexpected exception: " + e);
-           throw e;
-       } finally {
-           shutdown();
-       }
+            // Since this is the only client, lock should've been acquired, verify, block until condition is met
+            System.out.println("***** Wait until lock is acquired");
+            waitCondition = WaitConditionType.LOCK_ACQUIRED;
+            blockUntilWaitCondition.acquire();
+
+            int lockCount = 1;
+            assertThat(lockAcquiredObservables.get(clientId).getValue()).isEqualTo(lockCount);
+            assertThat(client.getLocks().get(lockId).getState().getType()).isEqualTo(LockStateType.HAS_LEASE);
+            assertThat(lockRevokedObservables.get(clientId).getValue()).isEqualTo(0);
+
+            waitCondition = WaitConditionType.NONE;
+
+            // Verify for 5 cycles that the lock is renewed
+            for (int i=0; i < RENEW_CYCLES; i++) {
+                System.out.println("***** Wait until lock is renewed");
+                // TODO: inspect the     // how many times the lease has been acquired by a different client
```

Comment:
Pavel are you sure the rebase was done correct? or is up to the latest? cause this had some changes I don't see reflected.

Reply:
Looks like I rebased before the changes to LockIT were made. I will rebase one more time.